pkg/api/v1/types.go

```diff
@@ -236,8 +236,8 @@ type VolumeSource struct {
 	// Glusterfs represents a Glusterfs mount on the host that shares a pod's lifetime.
 	// More info: http://releases.k8s.io/HEAD/examples/glusterfs/README.md
 	Glusterfs *GlusterfsVolumeSource `json:"glusterfs,omitempty"`
-	// PersistentVolumeClaimVolumeSource represents a reference to a
-	// PersistentVolumeClaim in the same namespace.
+	// PersistentVolumeClaim represents a reference to a
+	// PersistentVolumeClaimVolumeSource in the same namespace.
```

Comment:
PersistentVolumeClaimVolumeSource is an internal type name.  The comment is telling the user what this field does - it refers to a PersistentVolumeClaim object.  User's don't care what we name our mid-level structures, but PersistentVolumeClaim is a first-class API object.